core/src/main/scala/geotrellis/server/util/package.scala

```diff
@@ -0,0 +1,13 @@
+package geotrellis.server
+
+import java.io.{PrintWriter, StringWriter}
+
+package object util {
+  implicit class throwableExtensions[T <: Throwable](th: T) {
```

Comment:
It is helper function to retrieve the exception stack trace as a string, so the server would return smth more readble rather than i.e. an `IllegalArgumentException`.